modules/nodes-pods-autoscaling-custom-install.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-install_{context}"]
+= Installing the custom metrics autoscaler
+
+You can use the {product-title} web console to install the Custom Metrics Autoscaler Operator.
+
+The installation creates five CRDs:
+
+* ClusterTriggerAuthentication
+* KedaController
+* ScaledJob
+* ScaledObject
+* TriggerAuthentication
+
+.Prerequisites
+ifdef::openshift-origin[]
+* Ensure that you have downloaded the {cluster-manager-url-pull} as shown in _Obtaining the installation program_ in the installation documentation for your platform.
++
+If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red Hat Operators_.
+endif::[]
+
+* Uninstall the community KEDA, if present in your cluster. You cannot run both KEDA and the custom metrics autoscaler on the same {product-title} cluster.
+
+* Remove the KEDA 1.x custom resource definitions by running the following commands:
++
+[source,terminal]
+----
+oc delete crd scaledobjects.keda.k8s.io
+----
++
+[source,terminal]
+----
+oc delete crd triggerauthentications.keda.k8s.io
+----
+
+.Procedure
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose *Custom Metrics Autoscaler* from the list of available Operators, and click *Install*.
+
+. On the *Install Operator* page, ensure that the *All namespaces on the cluster (default)* option
+is selected for *Installation Mode*. This installs the Operator in all namespaces.
+
+. Ensure that the *openshift-keda* namespace is selected for *Installed Namespace*.  
+
+. Click *Install*.
+
+. Verify the installation by listing the Custom Metrics Autoscaler Operator components:
+
+.. Navigate to *Workloads* -> *Pods*.
+
+.. Select the `openshift-keda` project from the drop-down menu and verify that the `custom-metrics-autoscaler-operator-*` pod is running.
+
+.. Navigate to *Workloads* -> *Deployments* to verify that the `custom-metrics-autoscaler-operator` deployment is running.
+
+. Optional. Verify the installation in the {product-title} CLI using the following commands:
```

Comment:
* I think we use "Optional:" with a colon instead of a period.
* "{product-title} CLI" should just be "OpenShift CLI"